docs/src/examples/components/Avatar/Variations/AvatarExampleStatus.shorthand.tsx

```diff
@@ -1,21 +1,83 @@
 import React from 'react'
-import { Avatar } from '@stardust-ui/react'
+import { Avatar, StatusIndicator } from '@stardust-ui/react'
+
+const StatusIndicatorWrapper = (status: string) => {
+  switch (status) {
+    case 'Available':
+      return (
+        <StatusIndicator
+          icon={{ name: 'check', variables: { color: 'white', backgroundColor: 'green' } }}
+          title="Available"
+        />
+      )
+    case 'Away':
+      return (
+        <StatusIndicator
+          icon={{ name: 'clock', variables: { color: 'white', backgroundColor: 'yellow' } }}
+          title="Away"
+        />
+      )
+    case 'Busy':
+      return (
+        <StatusIndicator
+          icon={{ name: '', variables: { color: 'white', backgroundColor: 'red' } }}
+          title="Busy"
+        />
+      )
+    case 'DoNotDisturb':
+      return (
+        <StatusIndicator
+          icon={{ name: 'minus', variables: { color: 'white', backgroundColor: 'red' } }}
+          title="Do not disturb"
+        />
+      )
+    case 'BeRightBack':
+      return (
+        <StatusIndicator
+          icon={{ name: 'clock', variables: { color: 'white', backgroundColor: 'yellow' } }}
+          title="Be right back"
+        />
+      )
+    case 'Offline':
+      return (
+        <StatusIndicator
+          icon={{ name: '', variables: { color: 'white', backgroundColor: 'grey' } }}
+          title="Offline"
+        />
+      )
+    case 'PresenceUnknown':
+    default:
+      return <StatusIndicator title="Presence Unknown" />
+  }
+}
 
 const AvatarExampleStatusShorthand = () => (
   <div>
-    <Avatar src="public/images/avatar/small/matt.jpg" status="Available" />
+    <Avatar
+      src="public/images/avatar/small/matt.jpg"
+      status={StatusIndicatorWrapper('Available')}
+    />
     &emsp;
-    <Avatar src="public/images/avatar/small/matt.jpg" status="Busy" />
+    <Avatar src="public/images/avatar/small/matt.jpg" status={StatusIndicatorWrapper('Busy')} />
```

Comment:
Example code should be very concise as its primary purpose is to educate the user.  It is hard to grasp a concept quickly if it is too large or complex.  We also want to avoid making the user scroll up and down to see/edit the code when viewing the rendered example.

Issues
- This file is too many lines.
- Let's also remove the function call here for something more simple.  It looks like this could be done with an object instead, `statusIndicators.busy` should work.